ivadomed/scripts/automate_training.py

```diff
@@ -761,11 +761,11 @@ def automate_training(file_config, file_config_hyper, fixed_split, all_combin, p
                 combined_df = val_df
 
             results_df = pd.concat([results_df, combined_df])
-            results_df.to_csv(os.path.join(output_dir, "temporary_results.csv"))
-            eval_df.to_csv(os.path.join(output_dir, "average_eval.csv"))
+            results_df.to_csv(str(Path(output_dir, "temporary_results.csv")))
+            eval_df.to_csv(str(Path(output_dir, "average_eval.csv")))
 
     results_df = format_results(results_df, config_list, param_list)
-    results_df.to_csv(os.path.join(output_dir, "detailed_results.csv"))
+    results_df.to_csv(str(Path(output_dir, "detailed_results.csv")))
```

Comment:
Good use of the constructor for combining path + str. Good trick and readability. Too bad .to_csv hasn't introduced Path support yet :( from what I dug up.